tests/databases/molecule/fixtures/molecule_mongo_db.py

```diff
@@ -38,6 +38,17 @@
             molecule=stk.BuildingBlock('BrBr'),
             key={'SMILES': 'BrBr'},
         ),
+        CaseData(
+            database=stk.MoleculeMongoDb(
+                mongo_client=MockMongoClient(),
+                lru_cache_size=0,
+                jsonizer=stk.MoleculeJsonizer(
+                    key_makers=(stk.Smiles(),)
+                ),
+            ),
+            molecule=stk.BuildingBlock('BrBr'),
+            key={'SMILES': 'BrBr'},
+        ),
```

Comment:
I don't think we need a new test case for a molecule mongo db, since you didnt change that class.